src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -26,6 +38,29 @@ public class CommandPat
         private const string OutputOption = "--output";
         private const string OutputHelp = "How PAT information is displayed. [default: token]\n[possible values: none, status, token, base64, header, headervalue, json]";
 
+        private const string DomainOption = "--domain";
+        private const string DomainHelp = "The preferred domain used when acquiring Azure Active Directory access tokens. [default: microsoft.com]";
+
+        private const string AccessTokenPrompt = "AzureAuth ADO PAT";
+        private static readonly IEnumerable<AuthMode> AccessTokenAuthModes = new[] { AuthMode.Default };
+        private static readonly TimeSpan AccessTokenTimeout = TimeSpan.FromMinutes(15);
+
+        private static readonly string LockfilePath = Path.Combine(Path.GetTempPath(), AzureAuth.Ado.Constants.PatLockfileName);
+
+        // On all platforms the PAT cache should be in the same directory as a typical AzureAuth installation.
+        //   - On Windows this is usually `%LOCALAPPDATA%\Programs\AzureAuth`.

```

Comment:
You mentioned `usually`. What's our fallback if it's not found in the path you are expecting it to?

Reply:
I just mentioned "usually" because there are installations of AzureAuth that don't go to the normal directory. I can remove that word from this comment.

There isn't a fallback. This is where we expect it to be and if it's not there we can't guess at where else it should be.

Reply:
Technically we could see where the current executable is, but I think a better description is one Tucker told me that - the MSAL cache we use is also system wide. All versions of AzureAuth attempt to use the same cache to have the best experience, and that same concept applies here. Two different versions of AzureAUth across several different repos or a system install can all share the same PAT cache minimizing PAT creation notifications and potential auth prompts.